src/main/java/gregtech/common/terminal/app/worldprospector/WorldProspectorARApp.java

```diff
@@ -81,12 +93,48 @@ public AbstractApplication initApp() {
             }
             RectButtonWidget buttonWidget = new RectButtonWidget(x + i * 24, y + 18, 18, 18, 1);
             addWidget(new PhantomSlotWidget(handlers[i], 0, x + i * 24, y) {
+
+                @Override
+                public List<IGhostIngredientHandler.Target<?>> getPhantomTargets(Object ingredient) {
+                    if (!(ingredient instanceof ItemStack)) {
+                        return Collections.emptyList();
+                    }
+                    Rectangle rectangle = toRectangleBox();
+                    return Lists.newArrayList(new IGhostIngredientHandler.Target<Object>() {
```

Comment:
Do you know if this list gets appended to at all? If not, you could use `Collections.singletonList(etc)` instead.